app/src/main/java/org/mozilla/fenix/tabtray/TabTrayView.kt

```diff
@@ -656,21 +656,11 @@ class TabTrayView(
             String.format(view.context.getString(R.string.open_tab_tray_plural), count.toString())
         }
 
-        view.tabsTray.accessibilityDelegate = object : View.AccessibilityDelegate() {
-            override fun onInitializeAccessibilityNodeInfo(
-                host: View?,
-                info: AccessibilityNodeInfo?
-            ) {
-                super.onInitializeAccessibilityNodeInfo(host, info)
-                info?.let {
-                    info.collectionInfo = CollectionInfo.obtain(
-                        tabsAdapter.tabCount,
-                        1,
-                        false
-                    )
-                }
-            }
-        }
+        val isListTabView = view.context.settings().listTabView
```

Comment:
Do you think we could rename this function from `updateTabCounterContentDescription` to `updateViewAccessibility` or something? 
Also - could we possibly only call this function if `hasAccessibilityEnabled`?

Reply:
I think this should also be called with accessibility disabled, in case someone turns it on after it has opened the tabs tray.